Source/Optional.swift

```diff
@@ -59,6 +59,36 @@ public func -<< <T, U>(@noescape f: T -> U?, a: T?) -> U? {
 }
 
 /**
+    compose two functions that produce optional values, from left to right
+
+    - If the result of the first function is `.None`, the second function will not be inoked and this will return `.None`
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 121 characters (line_length)